src/pyper/env.py

```diff
@@ -0,0 +1,78 @@
+"""Load environment variables from a `.env` file."""
+import os
+from pathlib import Path
+
+_EQUALS = "="
+_NEW_LINE = "\n"
+
+_ENCLOSING_CHARS = f" \"'{_NEW_LINE}"
+
+
+class UnsupportedFileFormatError(Exception):
+    """Raised when the file format is not supported."""
+
+    def __init__(self, path: Path) -> None:
+        """Initialize the exception."""
+        super().__init__(f"Unsupported file format: {path.suffix()}")
+
+
+def auto_env(
+    *,
+    path: Path | None = None,
+    overwrite: bool = False,
+) -> dict[str, str]:
+    """Populate environment variables with variables from a `.env` file.
+
+    Parameters
+    ----------
+    path : Path | None, optional
+        Path to the `.env` file. If not set, `pyper` will try to find
```

Comment:
Should be a note that the .env file should be utf8

Reply:
Good idea!